### ISSUE TITLE
Chat keyboard navigation doesn't work, when focus switches to super-group

### ISSUE THREAD

**Issue by etoshapovalov** [overtake/TelegramSwift#1212]
When you're trying to navigate through chats, using keyboard shortcuts (Control+Tab and Control+Shift+Tab) and your focus stumbles upon a super chat (chat with various channels inside) - you can't progress further down with shortcuts. If you want to escape the chat window with a press of the escape button - the focus resets, and you need to start all over again. If you want to access chats that are next to your super-group, you'll need to use mouse to escape it, which defeats the whole purpose of using keyboard shortcuts in the first place.

Steps to reproduce:
 1. Click on any chat
 2. Press Control+Tab shortcut to jump to next chat
 3. Stumble upon a super-group

Expected behaviour:
After pressing control+tab, the focus should switch to a next chat in the list.
